infraestructure/services/http_calls.go

```diff
@@ -0,0 +1,44 @@
+package services
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func readResponseBody(resp *http.Response, url string) ([]byte, error) {
+	body, readErr := ioutil.ReadAll(resp.Body)
+
+	if readErr != nil {
+		log.Println("Error reading the response data from server in query ", url)
+		return nil, readErr
+	}
+
+	return body, nil
+}
+
+func makeHTTPCall(requestType string, url string, body io.Reader, queryParams map[string]string) (*http.Response, error) {
+
+	client := &http.Client{}
```

Comment:
Just curious about this, is there any performance issue when creating the client for every call?